examples/error-reporting/error_reporting.description

```diff
@@ -1,7 +1,7 @@
 // Ballerina does not have exceptions. Errors are reported by functions returning
-// error values.
+// `error` values.
 // `error` is its own basic type.
-// The return type of a function that may return an error value will be a union with error.
-// An error value includes a `string` message.
-// An error value includes stack trace from the point where `error(msg)` is called.
+// The return type of a function that may return an `error` value will be a union with `error`.
+// An `error` value includes a `string` message.
+// An `error` value includes stack trace from the point where `error(msg)` is called.
```

Comment:
from the point where the error is constructed? 🤔 @MaryamZi

Reply:
+1

Reply:
```suggestion
// An `error` value includes the stack trace from the point at which the error is constructed (i.e., `error(msg)` is called). 
```